09 Sum of letters.rb

```diff
@@ -0,0 +1,16 @@
+# Takes a string and if the sum of the position of every letter in the 
+# alphabet is even, returns true else false.
+def is_alpha(string)
+  sum = 0
+  # Uses ASCII values of each letter to calculate sum.
+  string.downcase.bytes do |x|
+    if x > 96 && x < 123
+      sum += x - 96
+    end
+  end  
+sum.even? ? true : false

```

Comment:
This alone will already return true or false so don't need what is on the right 

```suggestion
sum.even?

```